src/components/propositionalLogic/wiki.tsx

```diff
@@ -30,8 +30,12 @@ class WikiPropositionalLogic extends React.Component {
 				A&arr;B ist wahr genau dann, wenn entweder die Voraussetzung A
 				falsch ist oder A und B wahr sind. Ansonsten ist A&arr;B falsch.
 				A&harr;B ist wahr genau dann, wenn A und B beide falsch oder beide
-				wahr sind. Ansonsten ist A&harr;B falsch.
-			</p>
+				wahr sind. Ansonsten ist A&harr;B falsch.</p>
+				<h2>Literatur</h2>
+				Über die Unibibliothek gibt es beispielsweise folgende
```

Comment:
"beispielsweise" ist ein unsauberer Ausdruck für viele wissenschaftliche Situation. Man will Literatur eigentlich vollständig auflisten und keine stichprobenartige Selektion dieser geben und etwas vorenthalten. Also passender wäre etwas wie "Folgende Titel behandeln das Thema und können in der Unibibliothek als E-Book Gratis heruntergeladen werden."

Reply:
Aber Literaturangaben find ich immer super.

Reply:
Ich sehe das nicht so eng, aber gebe dir definitiv recht. War ein wenig sloppy von mir